code/modules/jobs/job_types/other/merc_classes/underdweller.dm

```diff
@@ -49,7 +49,7 @@
 
 	if(H.dna.species.id == "dwarf")
 		H.cmode_music = 'sound/music/cmode/adventurer/CombatOutlander2.ogg'
-		H.adjust_skillrank(/datum/skill/combat/axesmaces, 4, TRUE)
+		H.adjust_skillrank(/datum/skill/combat/axesmaces, 3, TRUE)
```

Comment:
why?

Reply:
truly makes no sense, we got multiple other mecs with expert in their skills , why is the dwarf one the only getting nerfed?